app/components/pagination.js

```diff
@@ -0,0 +1,3 @@
+import templateOnly from '@ember/component/template-only';
```

Comment:
I think on the current version of Ember this project is on we can completely omit this file and we'll get a template-only Glimmer component instead.